frontend/factory-ops/src/component/ItemList.jsx

```diff
@@ -4,14 +4,26 @@ import React from 'react';
 export default function ItemLists() {
```

Comment:
You can inject ItemList as parameter
``` js
const ItemList = ([{itemsInputList}] => {
    const [items, setItems] = useState([]);
    useEffect(() => {
        setItems(itemsInputList);
    })
}

```

or something like that ;)

Reply:
Probably yes, but I haven't learned anything like that yet